ui/server/src/main/resources/defaultUiConfig.conf

```diff
@@ -193,3 +193,7 @@ testDataSettings: {
   testDataMaxBytes: 200000
   resultsMaxBytes: 50000000
 }
+
+notifications {
+  duration: 24 hours
```

Comment:
I wonder why its exactly 24 hours ?

Reply:
"quite a long time, but not forever" :) do you have better value for default?